style.css

```diff
@@ -0,0 +1,71 @@
+:root{
+    --bg-color: #fffdd0;
+    --type-writer-speed: 5s;
+    --type-writer-no-of-characters: 24;
+}
+body{
+    margin: 0%;
+
+    display: grid;
+    place-content: center;
+    text-align: center;
+    min-height: 100vh;
+
+    background: var(--bg-color);
+}
+h1{
+    position: relative;
+    font-family: "Intern", sans-serif;
+    position: relative;
+
+    font-weight: 400;
```

Comment:
What's the default value of font-weight ?

Reply:
I think it's 400.

Reply:
right @Adishaikh Then y did you gave 400 if it is by default already set 400?